src/Messages/OWSMessageSender.m

```diff
@@ -607,11 +630,31 @@ - (void)groupSend:(NSArray<SignalRecipient *> *)recipients
     [self saveGroupMessage:message inThread:thread];
     NSMutableArray<TOCFuture *> *futures = [NSMutableArray array];
 
+    NSArray<NSString *> *blockedPhoneNumbers = _blockingManager.blockedPhoneNumbers;
+    int blockedCount = 0;
     for (SignalRecipient *rec in recipients) {
-        // we don't need to send the message to ourselves, but otherwise we send
-        if (![[rec uniqueId] isEqualToString:[TSStorageManager localNumber]]) {
-            [futures addObject:[self sendMessageFuture:message recipient:rec thread:thread]];
+        // We don't need to send the message to ourselves...
+        if ([[rec uniqueId] isEqualToString:[TSStorageManager localNumber]]) {
```

Comment:
Done. 

On a separate note, I've been thinking that we should add convenience accessors for some of these model objects.  In this case, I'd like to add `[SignalRecipient phoneNumberE164]` or perhaps `[SignalRecipient contactId]`.  We can discuss what's the best name, but the point is would be to distinguish whether or not we're looking for the database uniqueId or the phone number/signal account id.